src/Tizen.Sensor/Tizen.Sensor/Sensor.cs

```diff
@@ -541,9 +689,778 @@ private bool CheckSensorHandle()
+        /// </param>
+        /// <since_tizen> 6 </since_tizen>
+        public bool RecorderIsSupported(int type)
+        {
+            bool isSupported;
+            int error = Interop.SensorRecoder.RecorderIsSupported(type, out isSupported);
+            if (error != (int)SensorError.None)
+            {
+                Log.Error(Globals.LogTag, "Error checking if sensor recorder is supported");
+                throw SensorErrorFactory.CheckAndThrowException(error, "Sensor.RecorderIsSupported Failed");
+            }
+            return isSupported;
+        }
+
+        /// <summary>
+        /// Sets a recording option parameter.
+        /// </summary>
+        /// <param name="recorderOptionE">
+        /// Option parameter.
+        /// </param>
+        /// <param name="value">
+        /// Value.
+        /// </param>
+        /// <since_tizen> 6 </since_tizen>
+        public void RecorderOptionSetInt(int recorderOptionE, int value)
+        {
+            if (CheckRecorderOption())
+            { 
+                int error = Interop.SensorRecoder.RecorderOptionSetInt(recorderOption, recorderOptionE, value);
+                if (error != (int)SensorError.None)
+                {
+                    Log.Error(Globals.LogTag, "Error checking sensor recorder option set int");
+                    throw SensorErrorFactory.CheckAndThrowException(error, "Sensor.RecorderOptionSetInt Failed");
+                }
+            }
+            else
+            {
+                Log.Error(Globals.LogTag, "Recorder option is not created");
+            }
+        }
+
+        /// <summary>
+        ///  Sets an integer-type query parameter.
+        /// </summary>
+        /// <param name="recorderQueryE">
+        /// Query parameter.
+        /// </param>
+        /// <param name="value">
+        /// Value.
+        /// </param>
+        /// <since_tizen> 6 </since_tizen>
+        public void RecorderQuerySetInt(int recorderQueryE, int value)
+        {
+            if (CheckRecorderQuery())
+            { 
+                int error = Interop.SensorRecoder.RecorderQuerySetInt(recorderQuery, recorderQueryE, value);
+                if (error != (int)SensorError.None)
+                {
+                    Log.Error(Globals.LogTag, "Error checking sensor recorder query set int");
+                    throw SensorErrorFactory.CheckAndThrowException(error, "Sensor.RecorderQuerySetInt Failed");
+                }
+            }
+            else
+            {
+                Log.Error(Globals.LogTag, "Recorder query is not created");
+            }
+        }
+
+        /// <summary>
+        ///  Sets a time-type query parameter.
+        /// </summary>
+        /// <param name="recorderQueryE">
+        /// Query parameter.
+        /// </param>
+        /// <param name="time">
+        /// Time.
+        /// </param>
+        /// <since_tizen> 6 </since_tizen>
+        public void RecorderQuerySetTime(int recorderQueryE, int time)
```

Comment:
What is `E`?

Reply:
typedef enum
{	SENSOR_RECORDER_QUERY_START_TIME = 0, 
	SENSOR_RECORDER_QUERY_END_TIME, 
	SENSOR_RECORDER_QUERY_ANCHOR_TIME,
	SENSOR_RECORDER_QUERY_TIME_INTERVAL,
} sensor_recorder_query_e

I tried to replicate the variable names same as in C code.

Reply:
Do not use `E` suffix.
it is not c api

Reply:
E suffix removed.